epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.java

```diff
@@ -623,6 +656,41 @@ private void addEqualsLineForType(Builder builder, boolean useObjectHashCode, Ty
     }
   }
 
+  private void addEqualsLineForTypeUsingGetter(Builder builder, boolean useObjectHashCode,
```

Comment:
This could also be a method on the attribute info -> `attributeInfo.getAccessorCode()`

I think this approach could clean up the code in all the usages above too